homework_lesson_6/index_lesson_6_hw_store.html

```diff
@@ -0,0 +1,196 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title> base page </title>
+    <meta name="author" content="Anna Danielkiewicz">
+    <meta name="description" content="The base page we will be using for learning javascript">
+    <link rel="stylesheet" href="style_lesson_6_hw_store.css" type="text/css">    
+  </head>
+  <body>
+
+  <header> 
+    <nav class="main_navigation">
+        <ul>
+            <li><a href="#home">HOME</a></li>
+            <li><a href="#company">COMPANY</a></li>
+            <li><a href="#basepage" class="active">SHOP</a></li>
+            <li><a href="#blog">BLOG</a></li>
+        </ul>
+    </nav>
+  </header> 
+
+  <h1> WATCHES ONLINE STORE </h1>
+
+ <section>
+  <ul class="js-watches-list"></ul>
+  <h2 class="header2"></h2>
+ </section>
+
+  <script>
```

Comment:
Better keep your script as the last tag in body, thus under footer.

Reply:
I see it. I thought that I fixed this problem but after this comment  I look on my html through another browser - Everything is bigger and then I have the footer in the middle of the content - just as you describe it. But when I scroll to minimize the view, the footer is at the end. Hm I am thinking about it :) Thank you  Anna :)